open-metadata-resources/open-metadata-samples/access-services-samples/subject-area-client-samples/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/samples/SubjectAreaCatalogSample.java

```diff
@@ -93,92 +93,92 @@ private void run() throws SubjectAreaCheckedExceptionBase
          * create Organisation subject areas
          */
         String glossaryGuid =  glossary.getSystemAttributes().getGUID();
-        SubjectAreaDefinition organisation = createTopSubjectAreaDefinition(ORGANIZATION,glossaryGuid);
-        SubjectAreaDefinition hospital= createChildSubjectAreaDefinition(HOSPITAL,organisation,glossaryGuid);
-        SubjectAreaDefinition supplier= createChildSubjectAreaDefinition(SUPPLIER,organisation,glossaryGuid);
+        Category organisation = createTopCategory(ORGANIZATION,glossaryGuid);
```

Comment:
This sample is creating subject area definitions not categories - these renames of the types are incorrect